tests/3 - NavigationScenario.js

```diff
@@ -0,0 +1,13 @@
+description: "Navigating to an item of the list should show details, going back should keep the search.",
+
+steps: [
+  SearchBarComponent.searchFor(aahQuery),
+  AppComponent.firstResult(),
```

Comment:
This relies on `firstResultLink` in AppComponent. The `firstResult` function is probably inferred. 

I'm going to rename the link to `goToFirstResultLink`.